plants/species/Triticum_spelta/Triticum_spelta_references.md

```diff
@@ -0,0 +1,24 @@
+### References
+
+1.  [Shifting the limits in wheat research and breeding using a fully
+    annotated reference
+    genome](http://dx.doi.org/10.1126/science.aar7191).\
+    Rudi Appels, Kellye Eversole, Catherine Feuillet, Beat Keller, Jane
+    Rogers, Nils Stein\.... Hana imkov, Ian Small, Manuel Spannagl,
+    David Swarbreck, Cristobal Uauy. 2018. Science. 361
```

Comment:
Looks like there's something wrong with this reference. If [this](https://science.sciencemag.org/content/361/6403/eaar7191) is the article it refers to:
- it has lots more authors (need et al.?)
- the order of authors is different after Jane Rogers
- there's a strange ellipsis after Nils Stein, with a backslash that needs to be removed
- Hana imkov is Hana Šimková; we need to find out whether we support extended Latin alphabet or whether we want to write her name as Simkova
- do we actually use full names rather than initials in references?
- if there are too many authors to list, isn't the list normally abbreviated with an "et al." after the third author?

Reply:
@ens-ap5 can you advise?

Reply:
My citation knowledge is a little rusty, but I agree that the citation looks wrong.

Reply:
Thanks @azangru , this is what we currently have in the live site:
======================================== 
Shifting the limits in wheat research and breeding using a fully annotated reference genome.
Rudi Appels, Kellye Eversole, Catherine Feuillet, Beat Keller, Jane Rogers, Nils Stein.... Hana imkov, Ian Small, Manuel Spannagl, David Swarbreck, Cristobal Uauy. 2018. Science. 361
=========================================

Reply:
Since there are 30+ authors I used the ... as I sometimes see in other places. I am not aware of what format we use for these type of references

Reply:
Regarding Hana Šimková; and Nils Stein I can fix that

Reply:
And can also change it to et al.
I looked at a few examples in barley:
https://staging-plants.ensembl.org/Hordeum_vulgare/Info/Annotation/
And we use et al., but after more than 3 authors.
I can also start a discussion on Slack to see what others use

Reply:
@ens-ap5 Do you know if we support extended Latin alphabet? Can we handle names like Hana Šimková, or do we want Hana Simkova?

Reply:
> I can also start a discussion on Slack to see what others use

@gnaamati that would be helpful

Reply:
I think Hana Simkova is OK
I will follow up on the et al.

Reply:
I started a discussion, let's see what people say. Maybe you can also ask about Latin alphabet?

Reply:
1) As for the number of authors before et al that dependens of the citation style. Do we have a rule in Ensembl NV @nds @ens-gmaslen @andrewyatz ?

2) Regarding the text encoding, my first thought is we should support the same char sets supported by EuroPMC or PubMed. In my experience, when writing static content, I would copy and paste author names and the full citation from those websites

Reply:
Thanks Bruno, also started a thread on Slack

Reply:
test

Reply:
Hi @azangru, I updated the files, please take a look.
Regarding how to display references, it appears that there isn't consistency across Ensembl and that is being discussed. One option is to use EuroPMC codes for papers but I couldn't find one for the wheat paper.

Reply:
I also removed the backslashes, I assumed they were needed because that is how data was imported from Drupal.

Reply:
👍